tecken/apps.py

```diff
@@ -6,17 +6,55 @@
 
 import markus
 from django_redis import get_redis_connection
-import sentry_sdk
+from fillmore.libsentry import set_up_sentry
+from fillmore.scrubber import (
+    Scrubber,
+    Rule,
+    build_scrub_query_string,
+    SCRUB_RULES_DEFAULT,
+)
+from sentry_sdk.integrations.boto3 import Boto3Integration
 from sentry_sdk.integrations.django import DjangoIntegration
+from sentry_sdk.integrations.redis import RedisIntegration
 from sentry_sdk.integrations.logging import ignore_logger
 
 from django.conf import settings
 from django.apps import AppConfig
 
+from tecken.libdockerflow import get_release_name
+
 
 logger = logging.getLogger("django")
 
 
+SCRUB_RULES_TECKEN = [
+    # HTTP request bits
+    Rule(
+        path="request.headers",
+        keys=["Auth-Token", "Cookie", "X-Forwarded-For", "X-Real-Ip"],
+        scrub="scrub",
+    ),
+    Rule(
+        path="request.data",
+        keys=["csrfmiddlewaretoken", "client_secret"],
+        scrub="scrub",
+    ),
+    Rule(
+        path="request",
+        keys=["query_string"],
+        scrub=build_scrub_query_string(params=["code", "state"]),
+    ),
+    Rule(path="request", keys=["cookies"], scrub="scrub"),
```

Comment:
Because we're setting `send_default_pii=False`, there isn't any cookie information. So we probably don't really need this. Further, the cookie information is repeated `request.headers.Cookie` and this scrubbing doesn't account for that which is fine because it's not there anyhow on account of setting `send_default_pii=False`.

I'll remove this line.